tests/Benchmarks/GetAsciiStringBench.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using Xunit;
+using Microsoft.Xunit.Performance;
+using System;
+
+using System.Text;
+using System.IO.Pipelines.Text.Primitives;
+
+public class GetAsciiStringBench
+{
+    static int s_loops = 1000000; 
+
+    [Benchmark]
+    [InlineData("/plaintext")]
+    [InlineData("text/plain,text/html;q=0.9,application/xhtml+xml;q=0.9,application/xml;q=0.8,*/*;q=0.7")]
+    public static int GetAsciiString(string text)
+    {
+        string str;
+        int len = 0;
+        var bytes = (Span<byte>)Encoding.ASCII.GetBytes(text);
+        foreach (var iteration in Benchmark.Iterations) {
+            using (iteration.StartMeasurement()) {
+                for (int i = 0; i < s_loops; i++) {
+                    str = bytes.GetAsciiString();
+                    len += str.Length;
+                }
+            }
+        }
+        return len;
+    }
```

Comment:
Yes, you are right. Currently the API does not divide by the number of inner iterations, and it is something we will take a look at, but it will break current behavior (we might add it under a switch). In addition, we could add options to override this at runtime.